plugins/jamf/unit_test/test_get_device_groups.py

```diff
@@ -0,0 +1,53 @@
+import os
+import sys
+
+sys.path.append(os.path.abspath("../"))
+
+from unittest import TestCase, mock
+
+from icon_jamf.actions.get_device_groups import GetDeviceGroups
+from icon_jamf.actions.get_device_groups.schema import Input, Output
+from insightconnect_plugin_runtime.exceptions import PluginException
+from parameterized import parameterized
+
+from mock import (
+    STUB_CONNECTION,
+    STUB_DEVICE_ID,
+    mock_request_201,
+    mock_request_400,
+    mock_request_403,
+    mock_request_404,
+    mock_request_500,
+    mock_request_505,
+    mocked_request,
+)
+from util import Util
+
+
+class TestGetDeviceGroups(TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.action = Util.default_action_connector(STUB_CONNECTION, GetDeviceGroups())
+
+    @mock.patch("requests.request", side_effect=mock_request_201)
+    def test_erase_computer__ok(self, mocked_get) -> None:
```

Comment:
This may need renamed
```suggestion
    def test_get_device_groups_ok(self, mocked_get) -> None:
```